protocol/x/clob/keeper/clob_pair.go

```diff
@@ -186,7 +186,7 @@ func (k Keeper) setClobPair(ctx sdk.Context, clobPair types.ClobPair) {
 	b := k.cdc.MustMarshal(&clobPair)
 	store := prefix.NewStore(ctx.KVStore(k.storeKey), []byte(types.ClobPairKeyPrefix))
 	// Write the `ClobPair` to state.
-	store.Set(types.ClobPairKey(clobPair.GetClobPairId()), b)
+	store.Set(lib.Uint32ToBytes(clobPair.GetClobPairId().ToUint32()), b)
```

Comment:
Nit: In this file at least, seems like there's merit in keeping the function
```
// ClobPairKey returns the store key to retrieve a ClobPair from the index fields
func ClobPairKey(
	id ClobPairId,
) []byte {
	return lib.Uint32ToBytesForState(id.ToUint32())
}
```

Feels like trade-off between simplicity and readability. No strong preference